library/data/tags/src/testmod/java/org/quiltmc/qsl/tag/test/client/ClientTagsTestMod.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2021 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.tag.test.client;
+
+import net.fabricmc.api.ClientModInitializer;
+
+import net.minecraft.block.Block;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.item.Item;
+import net.minecraft.tag.Tag;
+import net.minecraft.text.LiteralText;
+import net.minecraft.util.Formatting;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.Registry;
+import net.minecraft.world.World;
+import net.minecraft.world.biome.Biome;
+
+import org.quiltmc.qsl.lifecycle.api.client.event.ClientWorldTickEvents;
+import org.quiltmc.qsl.resource.loader.api.ResourceLoader;
+import org.quiltmc.qsl.resource.loader.api.ResourcePackActivationType;
+import org.quiltmc.qsl.tag.api.TagRegistry;
+import org.quiltmc.qsl.tag.api.TagType;
+import org.quiltmc.qsl.tag.test.TagsTestMod;
+
+public final class ClientTagsTestMod implements ClientModInitializer {
+	/**
+	 * This tag is only registered on the client:
+	 * - worlds opened by the client must have this tag to work, only included in the quilt_tags_testmod:required_test_pack
+	 * data-pack so the testing can be exactly sure.
+	 * - can connect to servers that don't have the tag (since it's not TagType.CLIENT_SERVER_REQUIRED)
+	 */
+	public static final Tag<Block> TEST_REQUIRED_BLOCK_TAG = TagRegistry.BLOCK.create(
```

Comment:
Not sure what you mean?
The tag seems to be identified like the others.

Reply:
I think i5 means the field type (the others are `Tag.Identified`)

Reply:
Oh, that's because I don't display it's ID and content at world change since this one can be observed directly with F3.

so I don't need the restricted type, and can just get the most generic type.

Reply:
Wouldn't it be better to make it also identified just to be consistent with the other tags?

Reply:
It's identified now.